packages/backend/src/peer/service.ts

```diff
@@ -100,6 +101,10 @@ async function createPeer(
     return PeerError.InvalidStaticIlpAddress
   }
 
+  if (!isValidHttpUrl(options.http.outgoing.endpoint)) {
+    return PeerError.InvalidHTTPEndpoint
+  }
```

Comment:
This should also be checked (if specified) in `updatePeer`
https://github.com/interledger/rafiki/blob/5f34cc00d2c7c9a22e4b28266a84a61f351f87d7/packages/backend/src/peer/service.ts#L149